### PR TITLE
aii-opennebula: Include ONE labels support

### LINKED ISSUE #223: aii opennebula handle missing template
This error
```
[ERROR] Failed to execute plugin module's pxelinux Install method
[ERROR] Errors running plugin module pxelinux Install method: Can't call method "name" on an undefined value at /usr/lib/perl/AII/opennebula.pm line 650.
```
line 650 is the one with error in 
```perl
...
    if (@vmtpl) {
	$vmtemplate = $vmtpl[0];
        $main::this_app->info("Found VM template from ONE database: ", $vmtemplate->name);
    } else {
        $main::this_app->error("VM template is not available to be instantiated: ", $vmtemplate->name);
        return 0;
    }
...
```

### LINKED ISSUE #224: aii-opennebula: Include labels feature
This is a nice feature included since ONE 5.x, now is possible to include labels or tags to any resource (templates, running VMs, vnets...):

http://opennebula.org/opennebula-5-0-features-labels/

Only requires to include `LABEL`  variable within the ONE resource template, labels are included in the database and  it is also possible to make queries from the frontend.

### LINKED ISSUE #228: aii-opennebula: Include random passwords for SPICE/VNC connections
This is quite easy to implement, just requires to add `RANDOM_PASSWD="YES"`    within VM graphics sections template. It will include a random password each time that the VM is instantiated, this will improve the security.

### PR DESCRIPTION
**Pull request by alvarosimon**
This PR includes new LABELS feature available since ONE 5.x

 * Fix #223 : Handle missing template
 * Fix #224 : Include labels feature
 * Fix #228: Include SPICE/VNC random passwords

### CODE REVIEW

File: aii-opennebula/src/main/pan/quattor/aii/opennebula/schema.pan [hunk 1]
```diff
@@ -171,4 +171,9 @@ type opennebula_vmtemplate = {
     "diskcache" ? string with match(SELF, '^(default|none|writethrough|writeback|directsync|unsafe)$')
     "permissions" ? opennebula_permissions
     "pci" ? opennebula_vmtemplate_pci[]
+    @{labels is a string list to group the VMs under a given name and filter them
```
Comment (stdweird) [inline aii-opennebula/src/main/pan/quattor/aii/opennebula/schema.pan]: `string list` - >`list of strings`

File: aii-opennebula/src/main/pan/quattor/aii/opennebula/schema.pan [hunk 2]
```diff
@@ -171,4 +171,9 @@ type opennebula_vmtemplate = {
     "diskcache" ? string with match(SELF, '^(default|none|writethrough|writeback|directsync|unsafe)$')
     "permissions" ? opennebula_permissions
     "pci" ? opennebula_vmtemplate_pci[]
+    @{labels is a string list to group the VMs under a given name and filter them 
+    in the admin and cloud views. It is also possible to include in the list 
+    sub-labels using a common slash "Name, Name/SubName". This feature is available
```
Comment (stdweird) [inline aii-opennebula/src/main/pan/quattor/aii/opennebula/schema.pan]: example shoudl be pan code like `list("Name", "Name/SubName")`

**Comment by stdweird** [comment]
minor remarks, LGTM otherwise

**Comment by alvarosimon** [comment]
@stdweird remarks addressed

**Comment by stdweird** [comment]
LGTM

**Comment by stdweird** [comment]
@jrha ping

**Pull request merged**